content/en/docs/application-store/built-in-apps/radondb-postgresql-app.md

```diff
@@ -0,0 +1,94 @@
+
+4. Set a name and select an app version. Make sure RadonDB PostgreSQL is deployed in `demo-project` and click **Next**.
+
+   ![confirm-deployment](/images/docs/appstore/built-in-apps/radondb-postgresql-app/confirm-deployment.png)
+
+5. In **App Configurations**, you can use the default configuration or customize the configuration by editing the YAML file. When you finish, click **Deploy**.
+
+   ![set-app-configuration](/images/docs/appstore/built-in-apps/radondb-postgresql-app/set-app-configuration.png)
+
+6. Wait until RadonDB PostgreSQL is up and running.
+
+   ![RadonDB PostgreSQL-running](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-running.png)
+
+### Step 2: View PostgreSQL Cluster status
+
+1. On the **Overview** page of the project `demo-project`, you can see a list of resource usage in the current project.
+
+   ![project-overview](/images/docs/appstore/built-in-apps/radondb-postgresql-app/project-overview.png)
+
+2. In **Services** under **Application Workloads**, click the Service name of RadonDB PostgreSQL.
+
+   ![RadonDB PostgreSQL-service](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-service.png)
+
+3. In **Workloads** under **Application Workloads**, click the **StatefulSets** tab and you can see the StatefulSets are up and running.
+
+   ![statefulsets-running](/images/docs/appstore/built-in-apps/radondb-postgresql-app/statefulsets-running.png)
+
+   Click a single StatefulSet to go to its detail page. You can see the metrics in line charts over a period of time under the **Monitoring** tab.
+
+   ![statefulset-monitoring](/images/docs/appstore/built-in-apps/radondb-postgresql-app/statefulset-monitoring.png)
+
+4. In **Pods** under **Application Workloads**, you can see all the Pods are up and running.
+
+   ![pods-running](/images/docs/appstore/built-in-apps/radondb-postgresql-app/pods-running.png)
+
+5. In **Volumes** under **Storage**, you can see the PostgreSQL Cluster components are using persistent volumes.
+
+   ![volumes](/images/docs/appstore/built-in-apps/radondb-postgresql-app/volumes.png)
+
+   Volume usage is also monitored. Click a volume item to go to its detail page. Here is an example of one of the data nodes.
+
+   ![volume-status](/images/docs/appstore/built-in-apps/radondb-postgresql-app/volume-status.png)
+
+### Step 3: Access RadonDB PostgreSQL
+
+1. In **Pods** under **Application Workloads** page of the project `demo-project`.
+
+2. Click a single pod to expand the page to see container details.
+
+3. On the **Resource Status** page, click the **Terminal** icon.
+
+   ![RadonDB PostgreSQL-terminal](/images/docs/appstore/built-in-apps/radondb-postgresql-app/radondb-postgresql-terminal.png)
+
+4. In the displayed dialog box, enter commands in the terminal to use the app.
+
+   Run the following command, and then navigate to the password of the PostgreSQL cluster.
```

Comment:
I think this sentence can be combined with the preceding one. Besides, what does `navigate to the password of the PostgreSQL cluster` mean?

Reply:
merge the line